tests/test_cli.py

```diff
@@ -178,7 +178,8 @@ def test_format_argument(capsys, platform):
     # and we don't mind the CSV dialect, so just check call doesn't error
     read_csv = csv.Sniffer()
     # This gives '_csv.Error: Could not determine delimiter' if not a csv file
-    read_csv.sniff(csv_output, delimiters=",")
+    csv_output = csv_output.replace('\r', '')
```

Comment:
Is it possible to prepare the csv file in  way that needs us not to do this?

Reply:
This is because python got trouble sniffing csv generated from windows, I've already created an issue and pull request to python and it's in review. But I guess either we manipulate the test data to bypass the bug or avoid the bug from here.
https://github.com/python/cpython/issues/103925

Reply:
It's okay.